entity-service-impl/src/test/resources/response/nested_document.json

```diff
@@ -1,4 +1,101 @@
 {
+  "countries":
+  {
+    "valueList":
+    {
+      "values": [{"value": {"string": "india"}},{"value": {"string": "australia"}}]
+    }
+  },
+  "area": {
+    "valueList": {
+      "values": [ {
+        "value": {
+          "int": 1
+        }
+      },
+        {
+          "value": {
+            "int": 2
+          }
+        } ]
+    }
+  },
+  "timestamp_3": {
+    "valueList": {
+      "values": [
+        {
+          "valueMap": {
+            "values": {
+              "seconds": {
+                "value": {
+                  "long": 10
+                }
+              }
+            }
+          }
+        },
+        {
+          "valueMap": {
+            "values": {
+              "nanos": {
+                "value": {
+                  "long": 20
+                }
+              }
+            }
+          }
+        }
+        ]
```

Comment:
Please reformat the entire file once.
`spotlessApply` doesn't work on `json` files.